### PR TITLE
Fix: HR and Payroll Dashboard

### PR DESCRIPTION
**Pull request by Anurag810**
removed dashboard_fixture.py and added JSON for dashboard.

### CODE REVIEW

File: erpnext/hr/number_card/employees_left_(last_year)/employees_left_(last_year).json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+ "creation": "2020-07-22 11:56:32.947790",
+ "docstatus": 0,
+ "doctype": "Number Card",
+ "document_type": "Employee",
+ "dynamic_filters_json": "[[\"Employee\",\"status\",\"=\",\"frappe.defaults.get_user_default(\\\"Company\\\")\"]]",
```
Comment (nabinhait) [inline erpnext/hr/number_card/employees_left_(last_year)/employees_left_(last_year).json]: status should be static and the company should be dynamic filter.

File: erpnext/payroll/number_card/total_incentive_given(last_month)/total_incentive_given(last_month).json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+ "aggregate_function_based_on": "incentive_amount",
+ "creation": "2020-07-22 11:56:34.599047",
+ "docstatus": 0,
+ "doctype": "Number Card",
+ "document_type": "Employee Incentive",
+ "dynamic_filters_json": "",
+ "filters_json": "[[\"Employee Incentive\",\"docstatus\",\"=\",\"1\",false],[\"Employee Incentive\",\"payroll_date\",\"Timespan\",\"last year\",false],[\"Employee Incentive\",\"name\",\"=\",null,false]]",
```
Comment (nabinhait) [inline erpnext/payroll/number_card/total_incentive_given(last_month)/total_incentive_given(last_month).json]: Remove `name=null` filter

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/hr/number_card/employees_left_(last_year)/employees_left_(last_year).json (hunk 1 above)
Comment (Anurag810) [reply erpnext/hr/number_card/employees_left_(last_year)/employees_left_(last_year).json]: Done

File: erpnext/payroll/number_card/total_incentive_given(last_month)/total_incentive_given(last_month).json (hunk 2 above)
Comment (Anurag810) [reply erpnext/payroll/number_card/total_incentive_given(last_month)/total_incentive_given(last_month).json]: Done

**Review by nabinhait — APPROVED** [review]

**Pull request merged**